src/Components/Game/Cell.js

```diff
@@ -1,26 +1,25 @@
 import styles from "./Cell.module.css";
 
-const NewCell = (props) => {
+const Cell = (props) => {
   const givenCheck = props.value ? true : false;
-  const regex = /[^\d]/g;
 
   const blockInvalid = (event) => {
-    const blockedKeys = ['e','+','-','0'];
+    const blockedKeys = ['e','+','-','0','.'];
     if (blockedKeys.includes(event.key) || event.target.value) {
       event.preventDefault();
     }
   };
-
+  
   return (
     <input
       type="number"
       defaultValue={props.value}
       readOnly={givenCheck}
-      className={styles[`${props.type}-cell`]}
+      className={props.classList.map((thisClass) => styles[`${thisClass}`]).join(' ')}
```

Comment:
You don't need to use templated strings here, i.e. `thisClass => styles[thisClass]`